incubator/kubevirt/values.yaml

```diff
@@ -0,0 +1,21 @@
+kubevirt:
+  repository: "kubevirt"
+  tag: "v0.4.1"
+  pullPolicy: IfNotPresent
+  namespace: "kube-system"
```

Comment:
Remove the namespace here and from all resources. It can be specified upon installation and does not need any special configuration.

Reply:
done

Reply:
@unguiculus I can see that tests now are failing on
```
Error: release kubevir-10072-1 failed: ClusterRoleBinding.rbac.authorization.k8s.io "kubevir-10072-1-kubevirt-apiserver-auth-delegator" is invalid: subjects[0].namespace: Required value
```
Can you please specify how can I pass namespace upon installation?